src/main/java/vazkii/patchouli/client/book/template/component/ComponentCustom.java

```diff
@@ -37,6 +39,19 @@ public void onDisplayed(BookPage page, GuiBookEntry parent, int left, int top) {
 	public void mouseClicked(BookPage page, int mouseX, int mouseY, int mouseButton) {
 		callbacks.mouseClicked(mouseX, mouseY, mouseButton);
 	}
+	
+	@Override
+	public boolean shouldShowTooltip(BookPage page, int mouseX, int mouseY) {
+		return callbacks.shouldShowTooltip(mouseX, mouseY);
+	}
+	
+	@Override
+	public List<String> getTooltip(BookPage page, int mouseX, int mouseY) {
+		List<String> tooltip = callbacks.getTooltip(mouseX, mouseY);
```

Comment:
This logic seems questionable, shouldn't it maybe combine the two instead of ignoring the super?